.vsts.pipelines/builds/ci.yml

```diff
@@ -50,24 +50,6 @@ stages:
           submodules: true
         - template: /eng/common/templates/steps/source-build.yml
 
-      # Jobs that are useful to validate the build runs on the platform, but shouldn't run in the
```

Comment:
These change should be handled with a separate PR.  It provides better visibility and a cleaner history.